### DISCUSSION TITLE
[Feature Request] RTL SUPPORT

### DISCUSSION THREAD

**Discussion by MNBoy** [heroui-inc/heroui#1896] [Feature requests]
### Is your feature request related to a problem? Please describe.

I wanna use next ui for rtl website and the components don't have this feature, I add dir='rtl' for the body but it isn't work

### Describe the solution you'd like

I think you should add the direction property to the nextui config (ltr/rtl) so that we can change the direction of the components

### Describe alternatives you've considered

I think that idea should work because other systems UI has this feature (like ant design)

**Comment by jrgarciadev** [comment]
We are progressively adding RTL support

https://sb.nextui.org/?path=/story/components-breadcrumbs--default&globals=locale:ar-AE
https://sb.nextui.org/?path=/story/components-buttongroup--default&globals=locale:ar-AE
https://sb.nextui.org/?path=/story/components-tabs--static&globals=locale:ar-AE

https://github.com/nextui-org/nextui/releases/tag/%40nextui-org%2Freact%402.2.0

**Reply by MNBoy** [reply]
@jrgarciadev Thanks!

**Reply by e-Naeim** [reply]
please add rtl support for:
1- select (as the direction of the arrow and the text is not aligned correctly). 

2- labels, either select, inputs, or any component had a label, the label is not aligned correctly.
